### DISCUSSION TITLE
Feedback for Translation of English Discussions

### DISCUSSION THREAD

**Discussion by bigzoo** [community/community#4373] [Discussions]
Hello! :wave:
This is a discussion for gathering user feedback on the beta release of English translations for Discussions.

To learn more about this feature and how to use it, see the [changelog post](https://github.blog/changelog/2021-07-19-machine-translation-for-discussions-content/).

Please use this discussion to share your experience using the new feature, and any ideas you have on how the experience could be improved.

**Comment by TPS** [comment]
How/when can this be accessed via the official app?  [Will it be extended throughout?](https://github.com/github/feedback/discussions/3985)

**Reply by zeke** [reply]
Hi @TPS 👋🏼 

Thank you for your question. We are rolling out this feature as a beta to gather feedback and learn about whether this is useful for GitHub users. Depending on feedback and usage, we may expand the feature to other parts of GitHub's platform, like Issues or README files. When you say "official app", I assume you're referring to other parts of the GitHub.com website, but please correct me if you actually meant something like the GitHub Mobile app or GitHub Desktop.

**Reply by zeke** [reply]
I also left a comment over on the [GitHub Mobile feedback discussion](https://github.com/github/feedback/discussions/3985#discussioncomment-1073991) you mentioned here.

**Reply by TPS** [reply]
> When you say "official app", I assume you're referring to other parts of the GitHub.com website, but please correct me if you actually meant something like the GitHub Mobile app or GitHub Desktop.

@Zeke I _do_ mean the mobile apps.  ETA?

**Reply by zeke** [reply]
Thanks for the clarification. As I mentioned above, we are currently gathering feedback and usage data and will use that to determine if and when we'll expand machine translation features to other GitHub products. I recommend you give the current feature a try and let us know if you think it's useful.

**Reply by yassine-cherni** [reply]
It's good

**Reply by globalDEXcode** [reply]
natürlich ist das nützlich. ich kann leider kein englisch und solch eine funktion das automatisch jede sprache in meine landessprache übersetzen würde - in einer vernünftigen qualität - wäre sowas von goldwert. die frage ist eigentlich überflüssig :-D

**Comment by PythonCoderAS** [comment]
Woah this is really cool. Does it use Google Translate?

**Reply by qgustavor** [reply]
I tested some translations and looks like it uses Bing Translate. That's no surprise given the parent company is Microsoft.

**Reply by belcar-s** [reply]
Bing Translate is my favorite.

**Reply by Howlmaster** [reply]
That's cool.

**Comment by Anrock** [comment]
Non-english comments/discussions are pretty uncommon in my experience but this thing just saved me a couple of clicks. Would be cool if I can toggle it to be always on for any non-english content because next time, if there is no popup about it, I would probably forget that there's such a thing.

**Reply by adlai** [reply]
There should be strong warning prefixed to automatically translated text.

**Reply by commed551** [reply]
Erster Eindruck ! Sehr gut !

**Comment by interDist** [comment]
The origin language detection is buggy, as a result, the translation is useless.

**Reply by godalming123** [reply]
For example when someyone replied `no` it detected that as spanish because it didnt have a capital letter and offered to translate to english

**Reply by adlai** [reply]
It could perhaps be informed by the various language configurations of the account submitting the comment.

**Reply by coder0107git** [reply]
> For example when someone replied `no` it detected that as spanish because it didnt have a capital letter and offered to translate to english

I'm on this thread because of basically the exact same issue (`No` is capitalized in my case). I'm attaching a screenshot of where the problem occured.

<details>
<summary>Screenshot</summary><br>

**BTW**: The image is a link to the comment where I encountered this detection issue.

[![github translation incorrect language detection example](https://user-images.githubusercontent.com/76132013/228679199-bd5db21e-b44b-42b0-964c-b134864905d8.png)](https://github.com/orgs/community/discussions/49954#discussioncomment-5335875)

</details>

**Edit**:  Looks like this example has been mentioned in this discussion several times ([1](https://github.com/orgs/community/discussions/4373#discussioncomment-3024316), [2](https://github.com/orgs/community/discussions/4373#discussioncomment-3076183), [3](https://github.com/orgs/community/discussions/4373#discussioncomment-3292542), [4](https://github.com/orgs/community/discussions/4373#discussioncomment-3468783), [5](https://github.com/orgs/community/discussions/4373#discussioncomment-3736732), [6](https://github.com/orgs/community/discussions/4373#discussioncomment-4973789)) making this comment a little redudent.

**Reply by jsoref** [reply]
Hey, I just came here to write exactly about that instance! Thanks for referencing it 😄 

- https://github.com/community/community/discussions/49954#discussioncomment-5335875

**Reply by eliaselliotson** [reply]
I also had this bug when someone replied to a comment with "me too".  It translates to "I get all", which makes no sense.

**Reply by MasterInQuestion** [reply]
"Are there any specific guidelines for users to avoid triggering incorrect language detection?"
    This should be fixed from the service. Not coercing the users' writing habit.

**Reply by supervoidcoder** [reply]
> It could perhaps be informed by the various language configurations of the account submitting the comment.

How was this comment marked as disruptive lol it makes a good point, maybe the detection of the origin language could be aided using information from the commenter's profile, such as what language that user uses.

**Comment by PixelyIon** [comment]
This feature is extremely useful when moderating the discussions for large projects where some people might not be native English speakers but still want to participate while avoiding the hassle of externally translating what they said. If the comments could be translated without needing to press a button, it'd be even more useful. I'd imagine server load could be reduced if the translation is done automatically when commenting rather than later on for every instance.

**Reply by adlai** [reply]
Is it too much to hope that they _someday_ use some reputation-based system like Duolingo, instead of simply punting to the sibling corporation? [apparently, all translations are currently from Bing's automated translation; it's mostly reasonable, although no replacement for individual professional humans]

**Comment by dwisiswant0** [comment]
Always automatically translate to English and/or the user's preferred language _(and add the option to view originals)_ instead of having to click to translate.

**Reply by adlai** [reply]
> Always automatically _[...]_ _(and add the option to view originals)_

Your parenthesized comment is important, and that option must not be easily ignorable; the last thing social networks need is pointless arguments because two people think they speak the same language.

**Comment by SwiftyProgrammer690** [comment]
It has just been fixed! 
Read the blog post here: https://github.blog/changelog/2022-06-14-translations-for-discussions/

**Comment by osfanbuff63** [comment]
Amazing! Sounds like it will make project management so much easier!

**Reply by SwiftyProgrammer690** [reply]
It sure will!

**Comment by anaclumos** [comment]
Interesting, does it use some variant of Bing translate that ignores code quotes ``?

I previously researched on translating code-heavy documents but didn't have much progress because most translation providers also translate code quotes as well (for example, it translated `for variable in variables` even if it is a code block).

**Comment by bigint** [comment]
Hola mundo

**Comment by thanhcsf** [comment]
안녕하세요

**Comment by TaronVardanyan** [comment]
Hola mundo

**Comment by DatGamerYoloYT** [comment]
Sometimes English replies get translated and mostly the reason why is because of bad grammar. (Like I found in one community's discussions someone saying "no" and there being a translation and the translation only being "No" and it thinks it's spanish)

**Reply by NickCrews** [reply]
[Here is an example](https://github.com/duckdb/duckdb/discussions/4797#discussioncomment-4316135)

**Comment by ghost** [comment]
"No" **translate to english**

**Comment by alexlalves** [comment]
Can we please have an option to **disable** the option for Translation, or, even better, let **GitHub know which languages** I understand in my profile settings or so?

There are so many bilingual people out there. Why would I ever want a translation if I can perfectly understand the original message? It's just unnecessary visual clutter.

**Reply by osfanbuff63** [reply]
I agree, you should be able to say what languages you understand so that translations don't show then.

**Reply by paradoxicallist** [reply]
+1
I'd like to have this option too.

**Reply by adlai** [reply]
> There are so many bilingual people out there.

There are even companies specialising in helping you help the folks whose lingualities don't cover the entire conversation, although I'm not aware of Microsoft having acquired any such startup recently.

**Comment by IllagerCaptain** [comment]
![image](https://user-images.githubusercontent.com/76600137/182041966-a0705576-3d21-4a3b-8715-7e997d99d393.png)
![image](https://user-images.githubusercontent.com/76600137/182041959-3854fe9c-f805-442a-8231-1aca58f53794.png)
might wanna fix this, this word is both the same in English and Spanish, and they were speaking English, not Spanish.

**Reply by leeringrose** [reply]
What about detecting it as English by default if it's possible in terms of such commonly used words? Otherwise, the engine could look to translate it based on its database. What would occur if this logic goes through once?

**Reply by queenaliizaa** [reply]
How to Find [Goodwill Bins](https://goodwilloutletstore.com/) Near You
Goodwill operates outlet locations in many U.S. states, including California, Texas, Florida, and New York. To find one:

**Comment by CARDINNAL** [comment]
Because the other last viwe egine capacity of attention the x mark power
point presentation

**Comment by drop-nicholas** [comment]
https://github.com/orgs/community/discussions/30695#discussioncomment-3468773
This comment is not in Spanish, actually :smile: 
The engine seems a bit sensitive, and this is perfectly acceptable casual English.

**Reply by mcatanzaro** [reply]
Here's one where some English code followed by the `\o/` emoticon is misinterpreted as Portuguese: https://github.com/mesonbuild/meson/discussions/10808#discussioncomment-3638125

**Comment by kospl** [comment]
Дуже цікаво.

**Reply by adlai** [reply]
I think Github currently prioritizes Korean, Spanish, and Portuguese; that's what [their blog post](https://github.blog/changelog/2022-06-14-translations-for-discussions/) claims, without giving any reasoning for the notable lack of several other important languages.

**Comment by afkvido** [comment]
"No" is the same as "No", but gh thinks its in spanish and tries to translate to english

**Reply by TheAlexLichter** [reply]
Indeed 😅 Just witnessed it too. Same for "No."

**Reply by CARDINNAL** [reply]
many conquest are invited

**Reply by SwiftyProgrammer690** [reply]
why is this still alive?

**Comment by SwiftyProgrammer690** [comment]
off topic but, how is this thread still alive???

**Reply by afkvido** [reply]
no idea

**Reply by SwiftyProgrammer690** [reply]
me neither

**Reply by adlai** [reply]
> how is this thread still alive?

Not everyone eats the blog intravenously.

**Comment by SwiftyProgrammer690** [comment]
Off topic: This has been going on for more than 1 year... Why is this still not dead???

**Reply by afkvido** [reply]
💀

**Reply by canvural** [reply]
Why do you expect it to be dead? For me translations were just enabled today. So it means they are still actively working on it.

**Comment by obkduishinijf** [comment]
translated "no" to "No" 💀

**Comment by ghost** [comment]
It said that 'Ello' was spainish then translated it to 'It' lol

**Comment by jsoref** [comment]
It seems like the algorithm does something like:

```python
for lang in languages:
  if Bing.IsTextInLang(text, lang):
    return lang
```

The algorithm should be closer to:

```python
for lang in languages:
  if Bing.IsTextInLang(text, lang):
    # if the text is valid in the user's language, then assuming that it makes sense to offer to translate is silly
    # so, only offer to translate if it isn't valid in the user's language
    if not Bing.TranslateFrom(sourceLanguage: userLanguage, destLanguage: lang, text: text):
      return lang
```

In the short term, a number of tokens should be added as _stop words_, that includes [`no dice` (Spanish)](https://github.com/orgs/community/discussions/30695#discussioncomment-3468773) and [`no` (Spanish)](https://github.com/orgs/community/discussions/4373#discussioncomment-1979108) and [`o` (Portuguese apparently, although it'd also be valid Spanish)](https://github.com/mesonbuild/meson/discussions/10808#discussioncomment-3638125).

**Comment by pgorod** [comment]
I don't like it, sorry, where do I disable it?
Somebody answered "no" and it said "original answer in Spanish, do you want to translate to English", if I click that, it gives "No".
But I don't usually appreciate sites and browsers offering to translate. I know other people do. It just needs an easy way to be turned off completely.

**Comment by Khalarii** [comment]
Thinks hyperlinks are Spanish and wants to translate to English. Doesn't work well at all.

**Reply by Daxy7** [reply]
Yeah, here's an example in the wild, still not fixed months later
https://github.com/uBlockOrigin/uAssets/discussions/17361#discussioncomment-8691701

**Reply by jsoref** [reply]
That includes github issue links:
https://github.com/orgs/community/discussions/67181#discussioncomment-9787901

**Comment by kubo6472** [comment]
Surprisingly this just worked quite well with some Portuguese text.

**Comment by panva** [comment]
<img width="876" alt="Screenshot 2024-01-06 at 10 04 02" src="https://github.com/community/community/assets/241506/bbe37530-c8ca-454b-9389-a4e17a807fc6">
<img width="878" alt="Screenshot 2024-01-06 at 10 04 09" src="https://github.com/community/community/assets/241506/707dca04-7aa5-45b4-88b3-e43139889721">

'nuff said

**Comment by S-PScripts** [comment]
it translated no and said no was from spanish

**Reply by timoyates** [reply]
"Me too!!" was also identified as Spanish (Spanish translations simply showed the original English)

**Comment by MrAbade** [comment]
It is really helpful  :open_mouth:

**Comment by amidevous** [comment]
FRENCH

Serait-il possible d'avoir GitHub dans le langage choisi ?

Beaucoup de personnes dans le monde ne parlent pas l'anglais, ou mal.
Actuellement les gens se servent de google translate, qui traduit très très mal...

Avoir le choix de choisir son langage dans les paramètres de son compte serait un plus pour gérer l'interface complète.
Ensuite pour les discutions nous continuerons d'utiliser Google Translate (malheureusement qui fonctionne mal).

Si cette discussion recevait des PLUS PLUS avec AVIS alors peut être que GitHub apporterait cette possibilité.
:-)

Pour information Bitbucket et Gitlab sont traduit en Français alors que Github n'est pas traduit
  
  
  

ENGLISH

Would it be possible to have GitHub in the chosen language?

Many people in the world do not speak English, or badly.
Currently people use google translate, which translates very very badly...

Having the choice to choose your language in the settings of your account would be a plus to manage the whole interface.
Then for the discussions we will continue to use Google Translate (unfortunately that works badly).

If this discussion would receive PLUS PLUS with NOTICE then maybe GitHub would bring this possibility.
:-)

For information Bitbucket and Gitlab translate in French Github not translated

up

https://github.com/orgs/community/discussions/27869

**Comment by mahmudhmh** [comment]
One suggestion for improvement could be an option for users to provide feedback on the accuracy of translations. This could help fine-tune the system over time and ensure a more seamless communication experience for everyone. @bigzoo

**Comment by Wilsmac** [comment]
Hola mundo ✨👋🏻

**Comment by danieljarquin** [comment]
Providing feedback for translation of English discussions can vary depending on various factors such as the context, target audience, and the purpose of the translation. Here are some general pointers to consider when giving feedback:

Accuracy: Ensure that the translation accurately conveys the meaning and intent of the original English discussions. Check for any mistranslations, inaccuracies, or misunderstandings.

Clarity: The translation should be [clear](https://inglishe.com/idioms-for-clear/) and easy to understand for the target audience. Avoid overly literal translations that may confuse readers.

Consistency: Maintain consistency in terminology, style, and tone throughout the translation. Use consistent language and terminology to enhance readability and comprehension.

Naturalness: The translation should sound natural and fluent in the target language. Avoid awkward phrasing or unnatural expressions that may disrupt the flow of the text.

Cultural Sensitivity: Consider cultural nuances and sensitivities when translating discussions. Ensure that the translation is culturally appropriate and does not inadvertently offend or misrepresent cultural norms.

Formatting: Pay attention to formatting, punctuation, and layout to ensure that the translated text retains the structure and formatting of the original discussions.

Feedback Delivery: Provide specific feedback with examples to help the translator understand areas for improvement. Be constructive and encouraging in your feedback to facilitate learning and growth.

Contextual Understanding: Understand the context of the discussions and provide feedback that takes into account the specific context and purpose of the translation.

By focusing on these aspects, you can provide valuable feedback that helps improve the quality and effectiveness of the translation of English discussions.

**Comment by lelamaltez** [comment]
Providing feedback for translation of English discussions can vary depending on various factors such as the context, target audience, and the purpose of the translation. Here are some general pointers to consider when giving feedback:

Accuracy: Ensure that the translation accurately conveys the meaning and intent of the original English discussions. Check for any mistranslations, inaccuracies, or misunderstandings.

Clarity: The translation should be clear and easy to understand for the target audience. Avoid overly literal translations that may confuse readers.

Consistency: Maintain consistency in terminology, style, and tone throughout the translation. Use consistent language and terminology to enhance readability and comprehension.

**Reply by LiteBrite82** [reply]
Hi @lelamaltez , 

We’ve clarified our stance on using generative AI tools like ChatGPT within our Community via [this announcement](https://github.com/orgs/community/discussions/69006). Please review the guidelines to ensure your post meets them as failure to adhere to those rules can result in action taken by our moderator team.  You can read our updated [Code of Conduct](https://github.com/community/community/blob/baa5cd9020ef3f2bbf86dca1191d9c90f9b0822d/CODE_OF_CONDUCT.md#reasonable-use-of-ai-generated-content) and the [announcement](https://github.com/orgs/community/discussions/69006) for more details. Thank you for helping us maintain an authentic and beneficial space for everyone.

**Comment by leahanderson1** [comment]
"make a pr" was identified as portugese

**Comment by Mrgaton** [comment]
It is very useful thanks

**Comment by ljharb** [comment]
a comment "+1 for @ljharb" was mistakenly identified as being in Spanish, and "translate to english" of course didn't change anything.

in general, if the translation is identical to the original, you probably shouldn't show the CTA?

**Reply by hirehamir** [reply]
A similar thing happened at https://github.com/leptos-rs/leptos/discussions/2627#discussioncomment-9757193.

> https://leptos.dev/
> 
> ![image](https://private-user-images.githubusercontent.com/43425812/339158694-05fd5af1-06a8-46cc-a64e-81dcb8d78e88.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTgyMzQ4NTgsIm5iZiI6MTcxODIzNDU1OCwicGF0aCI6Ii80MzQyNTgxMi8zMzkxNTg2OTQtMDVmZDVhZjEtMDZhOC00NmNjLWE2NGUtODFkY2I4ZDc4ZTg4LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MTIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjEyVDIzMjIzOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWMxNjczMzJmZTQxMTM1NjZkOTZhYjQ5MjJjNmJkODRmYjgwNjIxYmYzNzE1MzhkYzQ1ZWExY2U2OWIzMTUwZGImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.MK1Pnt5uHZSqqfaQ_B4mzUFuCq3W494E7xinbpX_9Fg)

was mistakenly identified as being in Spanish, and "translate to English" didn't change anything.

**Reply by hirehamir** [reply]
![image](https://github.com/community/community/assets/43425812/c6764a18-0bac-4537-ac40-2065b11d3dbf)

**Reply by yaakov-h** [reply]
Same thing happened here:
https://github.com/dotnet/runtime/discussions/74213#discussioncomment-9865958

![image](https://github.com/community/community/assets/426009/6d6be4b2-dfb0-4b0e-b766-75850871f0fe)

**Comment by its-miroma** [comment]
Emoji are identified as Korean:

**Reply by its-miroma** [reply]
:heart:

**Reply by cheshire137** [reply]
I like to think love is just a Korean concept. 🤷🏻‍♀️ 😄 

![screenshot of a heart emoji comment interpreted as Korean](https://github.com/user-attachments/assets/57a8c5a0-d6c3-4f08-a958-1f7b7a4168d3)

**Comment by MasterInQuestion** [comment]
See: https://github.com/MasterInQuestion/Honorificabilitudinitatibus/blob/main/Translation%20Demo.txt
    ; on formatting design reference.

    In brief, in-view comparison shall be important:
    Lest unwieldy translation.

**Comment by DaoyuanLi2816** [comment]
It is very useful thanks

**Comment by autoSteve** [comment]
Adding a simple reference to a pull request is seen as Spanish, which is interesting, otherwise a brilliant feature.

<img width="380" alt="image" src="https://github.com/community/community/assets/37229860/d6bbbc9f-3055-4744-afb8-d19221c7d2fc">

**Reply by port19x** [reply]
Same for me with "No"

**Reply by candlerb** [reply]
Concrete example of "No" being treated as Spanish: https://github.com/netbox-community/netbox/discussions/17561#discussioncomment-10738664

**Comment by darpank** [comment]
Might have run into a bug, 
- seeing the Translate to English link even though the text is already translated. 
- clicking on the link changes the link to View Original but the text remains the same. 
- subsequently clicking on the link just toggles the link with no change to the text

<img width="940" alt="image" src="https://github.com/user-attachments/assets/42f76f15-3168-47ab-a79f-6265d270c20a">

Can be see in this discussion comment https://github.com/orgs/community/discussions/139936#discussioncomment-10821051

Seen only on this comment, the other Spanish comment on this discussion seems to be toggling fine. A refresh resolved the issue but i was able to capture the behavior in a video that is too large to attach here but can provide it offline.

**Reply by darpank** [reply]
Just realized what i did to cause this. These are the steps to repro

1. Find a comment in Spanish
<img width="897" alt="image" src="https://github.com/user-attachments/assets/3d065cc7-1b3f-4bc6-bbdd-3bbb48052407">

2. Click Translate To English
<img width="905" alt="image" src="https://github.com/user-attachments/assets/25418329-1e36-4b65-be8f-ed0dd78beb5c">

3. Click Give Feedback, that brings you to this discussion. Then click browser return. Now you are in the bad state where the text is always in English and the links just toggle when you click, until you Refresh. I am using Arc for Mac (Chromium)
<img width="891" alt="image" src="https://github.com/user-attachments/assets/2129c680-778f-4f9b-bf00-776942c332c1">

**Reply by S-PScripts** [reply]
Same thing happens on my chromebook!

**Comment by MasterInQuestion** [comment]
Seems it has rather limited support for languages:
    https://github.com/orgs/community/discussions/139005#discussioncomment-10729453 (Portuguese)
    https://github.com/orgs/community/discussions/139005#discussioncomment-10850326 (Arabic)
    https://github.com/orgs/community/discussions/142405 (Persian)
    https://github.com/orgs/community/discussions/88425#discussioncomment-8185519 (Hebrew)
    https://github.com/orgs/community/discussions/126301#discussioncomment-10383838 (Bengali)
    https://github.com/orgs/community/discussions/139005#discussioncomment-10722284 (Chinese)
    https://github.com/orgs/community/discussions/139005#discussioncomment-10930571 (German)
    https://github.com/orgs/community/discussions/135572#discussioncomment-10986883 (Swedish)
    https://github.com/orgs/community/discussions/134410#discussioncomment-10230646 (Polish)
    https://github.com/orgs/community/discussions/67196#discussioncomment-9436745 (Turkish)
    [ https://github.com/jarvis394/geekr/discussions/145](https://github.com/jarvis394/geekr/discussions/145) (Russian)
    https://github.com/orgs/community/discussions/4373#discussioncomment-3728017 (Ukrainian)

    And likely didn't work for sub-comments:
    https://github.com/orgs/community/discussions/139005#discussioncomment-10695878 (Vietnamese)
    https://github.com/orgs/community/discussions/139005#discussioncomment-10843125 (French)
    .
    It does, but seemingly...
    https://github.com/orgs/community/discussions/52325#discussioncomment-5565799

**Reply by MasterInQuestion** [reply]
サポート一介 Portuguese, Spanish, Korean..?
    https://github.com/orgs/community/discussions/4373#discussioncomment-4562737

**Reply by MasterInQuestion** [reply]
La differenza entre instrumento finemente e implemento embotado...

    Spanish not detected:
    https://github.com/orgs/community/discussions/139005#discussioncomment-10875739

    Also no UI to activate on hidden comment:
    https://github.com/orgs/community/discussions/142199#discussioncomment-11094233 (Spanish detected)

**Comment by webbrowser11** [comment]
Spanish was detected when english: "No..."

**Comment by zc-devs** [comment]
![Screenshot 2024-11-28 125628](https://github.com/user-attachments/assets/a6c67d73-639f-49ec-9201-039128c4c818)
![Screenshot 2024-11-28 125611](https://github.com/user-attachments/assets/97478440-f8b1-4592-b921-de4f026f64bf)

No, this translation was useless.

---

How to disable automatic translations?

**Reply by webbrowser11** [reply]
¯\_(ツ)_/¯

**Reply by rhodes-b** [reply]
https://github.com/ghostty-org/ghostty/discussions/5759#discussioncomment-12353811

this just happened to me clearly there is an issue with the word "no"

**Comment by webbrowser11** [comment]


**Reply by webbrowser11** [reply]
❤️🍜

**Comment by EngrPips** [comment]
The translation makes sense.

**Comment by S-PScripts** [comment]
When will you fix "No" being translated to Spanish??

**Reply by S-PScripts** [reply]
erm... fix it!

**Comment by peppy** [comment]
![Safari 2025-04-08 at 06 16 38](https://github.com/user-attachments/assets/95e5b136-fb63-4a0e-9777-b8f82e028bf0)

**Reply by EngrPips** [reply]
What does this mean?

**Reply by peppy** [reply]
started appearing on every "no", you tell me 😅😅

**Reply by EngrPips** [reply]
I think you can ignore it.

**Reply by peppy** [reply]
Yes. I can also post the feedback in this thread like I was instructed to by the "Give Feedback" link. I intentionally posted a screenshot showing it happening alongside countless similar reports dating back years. There's a hope I have that this would re-raise the issue with the github team, but based on lack of action over the years it's a shot in the dark.

**Reply by EngrPips** [reply]
Yeah, but raising it is good, and maybe it will be fixed later, if not sooner.

**Comment by jackyzimu** [comment]
Where exactly can I click on the menu(o_ _)ﾉ

**Reply by EngrPips** [reply]
🤔

**Comment by jay-ramani** [comment]
While it's the same in Spanish, why is GitHub wrongly detecting a English language response and conversation in https://github.com/zufuliu/notepad4/discussions/997#discussioncomment-12803123?

**Comment by ghost** [comment]
Same as https://github.com/orgs/community/discussions/4373#discussioncomment-12887516 try starting a comment here with - ah!


Of course it won't work when I want to show it.

**Comment by S-PScripts** [comment]
**WHY DOES GITHUB STILL TRANSLATE NO (SINCE IT THINKS IT'S SPANISH)? IT'S BEEN LIKE THIS FOR A YEAR**

**Reply by S-PScripts** [reply]
no

**Comment by eamonburns** [comment]
It thinks that "TempleOS" is Spanish, but doesn't change anything when I click to translate to English

**Reply by eamonburns** [reply]
TempleOS

**Comment by chylex** [comment]
First time seeing this feature:

![image](https://github.com/user-attachments/assets/3f535e3e-ee83-4c33-8588-71ea217600dc)

**Comment by wazeerc** [comment]
Muito preciso 😂

![image](https://github.com/user-attachments/assets/65dfcec9-4d53-42af-bf51-8672cb0b5c28)

**Comment by loloide** [comment]
be careful when translating spanish
although "por qué" is "why", "porque" means "because".

I just translated "porque" and gave me "why"

**Comment by daiyam** [comment]
My `no` comment was flagged as spanish...

**Comment by zachlewis** [comment]
Here's a strange (but harmless) one:

https://github.com/AcademySoftwareFoundation/OpenImageIO/discussions/4787#discussioncomment-13959457

"I'll do my best!" is falsely identified as Portuguese!

**Reply by Julian-Dumitrascu** [reply]
That's "artificial intelligence" for you!

**Reply by 1mike12** [reply]
A Microsoft é uma empresa pequena, mas está fazendo o seu melhor.

**Comment by meator** [comment]
<img width="297" height="157" alt="image" src="https://github.com/user-attachments/assets/f6872975-7383-4171-8d1f-1d89157c3b06" />

<img width="310" height="190" alt="image" src="https://github.com/user-attachments/assets/361f8ce5-ad8d-41e1-ac1f-0f2c39494f21" />

**Comment by MrHinsh** [comment]
1. I wrote in English, using a machine and browser both set to English. Why was my input detected as Spanish? https://github.com/nkdAgility/azure-devops-migration-tools/discussions/3054

If its because im currently in Mexcio, then thats a false assertion that it should be spanish?

2. This is a poor way to collect feedback. I almost didn’t submit mine, and I imagine many others won’t bother to scroll down.

**Reply by supervoidcoder** [reply]
It actually has nothing to do with your machine or browser or location, it's just the text (at least for now). For some reason, it just always translates "no" to english because it thinks it is spanish. Weird, but hasn't been fixed in over a year.

**Reply by jsoref** [reply]
- or two, but who's counting? https://github.com/community/community/discussions/4373#discussioncomment-5471087

**Reply by fredsh2k** [reply]
diff example, same detection of spanish for some reason
https://github.com/github/delivery-org/discussions/4912#discussioncomment-14914082

**Comment by enridami** [comment]
cool, buts be careful

**Comment by 1mike12** [comment]
I thought this was a new feature but can't believe this started in 2021?? 

https://github.com/MonitorControl/MonitorControl/discussions/1595#discussioncomment-10471700

There needs to be a minimum word count for translations, or at least some more rules that if the previous comments by a user in the thread were all in language X that you do not assume language Y. 

Or get a different model that when they language detection score is low due to very short samples, to not translate. 

Ie one word comments like NO, or TODO should not be identified as Spanish

**Comment by danielzgtg** [comment]
Not working at https://github.com/FPGAwars/icestudio/discussions/843#discussioncomment-11954102 . Original comment is Spanish, but I still see no English even before/after clicking "Translate to English".

<img width="487" height="213" alt="Stuck in Spanish" src="https://github.com/user-attachments/assets/2ef0437d-9f27-4e1f-a0da-d377b34793db" />

I see an error in DevTools:

<img width="1351" height="125" alt="can't access property translations intermediate value 0 is undefined" src="https://github.com/user-attachments/assets/90329ca8-2f36-4565-a741-28f2ba6ccd7e" />

**Comment by Virakal** [comment]
When I try to use this the translation gets rejected with a 410 error, containing the following:

`error | "This translation service has been deprecated and is no longer available. Please use alternative translation methods."`

**Comment by eamonn-webster** [comment]
Post is in Portuguese, I click "Translate to English". No English translation is shown, though I did get a link to give feedback.
